reader.go

```diff
@@ -73,8 +73,13 @@ func NewReader(input io.ReaderAt, options ...ReaderOption) *Reader {
 		panic(err)
 	}
 
-	column := f.Root()
-	schema := NewSchema(column.Name(), column)
+	isEmpty := f.Size() == 0
+
+	var schema *Schema
+	if !isEmpty {
+		column := f.Root()
+		schema = NewSchema(column.Name(), column)
+	}
```

Comment:
If I'm reading this correctly, it's meant to allow zero-size files? Should we allow it considering zero-size files aren't valid parquet files?

Reply:
Should we panic instead ?

Reply:
switched to a panic